pom.xml

```diff
@@ -47,11 +47,6 @@
             <name>JBoss QA snapshot repository</name>
             <url>http://${env.NEXUS_HOST}/nexus/content/repositories/jboss-qa-snapshots</url>
         </snapshotRepository>
-        <repository>
-            <id>jboss-qa-releases</id>
-            <name>JBoss QA release repository</name>
-            <url>http://${env.NEXUS_HOST}/nexus/content/repositories/jboss-qa-releases</url>
-        </repository>
```

Comment:
I have to remove it because:

1. Two repository tags are invalid (it'll fail any mvn command)
1. We wanna push to upstream.

I may remove the snapshot as well.